AlertAction.py

```diff
@@ -0,0 +1,27 @@
+import AlertAction_definitions as definitons 
+
+def get_Alertaction(alertTarget, breachType):
+  alertAction = None
+  if alertTarget == 'TO_CONTROLLER':
+      send_to_controller(breachType)

```

Comment:
return value is not captured anywhere. Is the return value from method necessary?

Reply:
I have tried to introduce return value in this function to check invalid alertTarget 

for valid targets, I used a dictionary 
"targetType = {'TO_CONTROLLER': 0 , 'TO_EMAIL': 1}" in AlertAction_definitions.py ,Line 5 

Can you please suggest if this approach is okay?